creator-node/src/services/stateMachineManager/stateReconciliation/index.js

```diff
@@ -135,7 +166,7 @@ class StateReconciliationManager {
         JOB_NAMES.ISSUE_MANUAL_SYNC_REQUEST,
         job,
         handleSyncRequestJobProcessor,
-        logger,
+        manualSyncLogger,
         prometheusRegistry
       )
```

Comment:
good job changing the logger. we also want to change this one to be on the manualSyncQueue -- so `manualSyncQueue.process` instead of `reconciliationQueue.process`

Reply:
It already is

Reply:
ah I see it above now, nice